src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -326,10 +333,19 @@ public void addRow(T item, List<Content> contents) {
             HtmlStyle cellStyle = columnStyles.get(colIndex);
             // Always add content to make sure the cell isn't dropped
             var cell = HtmlTree.DIV(cellStyle).addUnchecked(c.isEmpty() ? Text.EMPTY : c);
+            boolean matchFound = c.isEmpty() || checkFormElements.matcher(c.toString()).find();
             cell.addStyle(rowStyle);
+            if (!matchFound) {
+                cell.put(HtmlAttr.ROLE, "tablist")
+                    .put(HtmlAttr.TABINDEX, "0");
+            }
 
             for (String tabClass : tabClasses) {
                 cell.addStyle(tabClass);
+                if (!matchFound) {
+                    cell.put(HtmlAttr.ROLE, "tablist")
+                        .put(HtmlAttr.TABINDEX, "0");
+                }
```

Comment:
These lines are redundant; see lines 338-341.

Reply:
Yes Keith, here we have 2 styles.
1. rowStyle without tabClasses : where div is `<div class="col-first even-row-color">` 
2. rowStyle with tabClasses : where div is <div class="col-last odd-row-color all-classes-table all-classes-table-tab2">

For both the styles tab should get focus, to do that we need to add `role=tablist tabindex=0`.